qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -63,7 +61,7 @@ def __init__(
         self,
         qubits: Sequence[int],
         lengths: Iterable[int],
-        backend: Optional[Backend] = None,
+        backend: Backend = None,
```

Comment:
You cannot override this policy because this is a breaking API change. Seems like you just want to know the basis gate. If backend is not specified, you don't need to transpile because in principle we cannot compile any code without target.

Reply:
I think this is a mistake. I am not sure why I did this. I will revert it back.